### ISSUE TITLE
bug/IndexError: list index out of range in partition()

### ISSUE THREAD

**Issue by TekeDado** [Unstructured-IO/unstructured#2417]
**Describe the bug**
When the `partition` func uses the  hi-res strategy with default model (a little hard to parse from the docs, but i think yolox?) and tries to guess table html, it raises `IndexError: list index out of range` when it gets to the line `prediction = recognize(outputs_structure, x, tokens=ocr_tokens)[0]` of run_prediction, which appears to occur when trying to read `tables_agent.predict(cropped_image, ocr_tokens=table_tokens)`. this does **not** occur if i instead specify `detectron2_onnx` as the model_name param to partition(). This bug was also noted over [here](https://github.com/Unstructured-IO/unstructured-inference/issues/306).

**To Reproduce**
```
elements = partition(
    url=pdf_url,
    include_page_breaks=False,
    content_type="application/pdf",
    skip_infer_table_types=[],
    pdf_infer_table_structure=pdf_infer_table_structure,
)
```

**Environment Info**
This is using unstructured 0.12.0, the latest release. 

**Additional context**

**Comment by MthwRobinson** [comment]
Thanks @TekeDado. Opened https://github.com/Unstructured-IO/unstructured-inference/pull/317 on the `unstructured-inference` side to address.
